src/sentry/static/sentry/app/views/issueList/actions.jsx

```diff
@@ -563,10 +564,22 @@ const IssueListActions = createReactClass({
             </GraphHeader>
           </GraphHeaderWrapper>
           <EventsOrUsersLabel className="align-right">
-            <ToolbarHeader>{t('Events')}</ToolbarHeader>
+            <React.Fragment>
+              {t('Events')}
+              <StyledQuestionTooltip
+                title={t('Number of events since the issue was created')}
+                size="xs"
+              />
+            </React.Fragment>
           </EventsOrUsersLabel>
           <EventsOrUsersLabel className="align-right">
-            <ToolbarHeader>{t('Users')}</ToolbarHeader>
+            <React.Fragment>
+              {t('Users')}
+              <StyledQuestionTooltip
+                title={t('Unique users affected since the issue was created')}
+                size="xs"
+              />
+            </React.Fragment>
```

Comment:
Are these `React.Fragments` needed? It has a parent container so doesn't seem necessary. See https://reactjs.org/docs/fragments.html